schema.sql

```diff
@@ -1,2 +1,2 @@
 CREATE DATABASE vet_clinic;
-CREATE TABLE animals(id INT GENERATED ALWAYS AS IDENTITY, name VARCHAR(250), date_of_birth DATE, escape_attempts INT, neutered BOOLEAN, weight_kg DECIMAL(5,2), PRIMARY KEY(id));
\ No newline at end of file
+CREATE TABLE animals(id INT GENERATED ALWAYS AS IDENTITY, name VARCHAR(250), date_of_birth DATE, escape_attempts INT, neutered BOOLEAN, weight_kg DECIMAL(5,2),species VARCHAR(250), PRIMARY KEY(id));
```

Comment:
- [ ] @shaaibu7 great work adding the species column, but you did it wrong we are supposed to add it using the `ALTER TABLE` command, this is due to the assumption that the table is already created and is in a database which would require us to use SQL command to make any changes to the table. Kindly remove the manually added column and add it using the `ALTER TABLE` command.